DataStructure/src/main/list/ResizableArray.java

```diff
@@ -0,0 +1,164 @@
+package main.list;
+
+/**
+ * Creates a Resizable List of Integer elements like the ArrayList Implementation in Java
+ */
+
+public class ResizableArray {
+    private int size;
+    private int indexOfLastElement;
+    private int list[];
+
+    /**
+     * @param size
+     * Takes in the size of the list.
+     * If left blank default size of 50 will be used
+     */
```

Comment:
constructor description - Constructs a resizable array
parm description - size of the list
remove if blank line.